src/pages/components/avatar.mdx

```diff
@@ -0,0 +1,105 @@
+---
+title: Avatar
+description: >
+  Avatars display typography, icons, or images in different sizes. They help users quickly recognize
+  an entity.
+package: '@zendeskgarden/react-avatars'
+propSheets:
+  - avatars/src/elements/Avatar.tsx
+---
+
+<Usage>
+  <UsageSection type="use">
+    <ul>
+      <li>To briefly explain the function of a user interface element</li>
+      <li>To give contextual help or information regarding ambiguous actions</li>
+    </ul>
+  </UsageSection>
+  <UsageSection type="misuse">
+    <ul>
+      <li>To display actions vital to using the UI</li>
+      <li>To display truncated text a native `title` attribute should be used</li>
+    </ul>
+  </UsageSection>
+</Usage>
+
+## Examples
+
+### Shape
+
+An Avatar can be a circle or rounded square. View the shape usage best practices.
```

Comment:
Link to "best practices"?

Reply:
For links that don't exist yet, we can have a local hyperlink `[Foobar](#)`. We will just need to clean these up before a wider release.

Reply:
Updated with a placeholder